pyproject.toml

```diff
@@ -58,5 +58,4 @@ exclude = '''
 [[tool.poetry.source]]
 name = "pypi-test"
 url = "https://test.pypi.org/simple/"
-default = false
-secondary = true
\ No newline at end of file
+priority = "explicit"
```

Comment:
Is this to work with a later version of Poetry? If so, can we recommend using a version of poetry in CONTRIBUTING.md?

Reply:
I don't know what exactly was barking at me. It came from the commit hooks.

Reply:
Actually, let's just drop pypi-test from the sources, it's not needed

Reply:
Removed it myself...